### ISSUE TITLE
Unable to /place anything from the mod

### ISSUE THREAD

**Issue by AvaAsUsual** [Faboslav/it-takes-a-pillage#1]
### Minecraft version information

1.21.1

### Mod loader information

Fabric

### Mod loader version information

0.16.9 Fabric

### Mod version information

fabric-mc1.21.1-1.0.4

### Expected Behavior

I expected for the /place structure command to work as it usually does and place the structure down

### Actual Behavior

After doing the command "/place structure takesapillage:pillager_camp", "failed to place structure" appears in chat

### Reproduction Steps

Join any world with only "It Takes a Pillage Continuation" on
Type "/place structure takesapillage:(either structure)"
See that "failed to place structure" appears in chat

### Screenshots & files

![image](https://github.com/user-attachments/assets/d1c6cc7b-c131-4c82-ab71-04648b8ce36f)

**Comment by Faboslav** [comment]
Hello, i think that this is happening due to extra flatness check.

**Comment by Faboslav** [comment]
Yeah, i just checked, this is the case, closing this issue, terrain needs to be very flat, so the structure can actually spawn.
